applications/FSIApplication/tests/convergence_accelerator_spring_MPI_test.py

```diff
@@ -166,7 +168,7 @@ def test_accelerator(self,accelerator_settings,force1,force2,solution):
 
         while (nl_it <= self.accelerator_iterations):
 
-            print(mpi.rank,": Iteration: ", nl_it," residual norm: ", res_norm, file=sys.stderr)
+            print(KratosMultiphysics.DataCommunicator.GetDefault().Rank(),": Iteration: ", nl_it," residual norm: ", res_norm, file=sys.stderr)
```

Comment:
better use logger then you don't have to check the rank

Reply:
I think that in this case we wanted to print the rank together with the residual norm. Using the Logger would imply to only print the rank 0. Am I wrong?

Reply:
in this case: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python/add_logger_to_python.cpp#L181
;)

Reply:
:heart_eyes: